src/redux/initialState.js

```diff
@@ -13,7 +13,12 @@ import actions from 'actions';
 
 export default {
   viewer: {
-    disabledElements: {},
+    disabledElements: {
+      thumbnailControl: {
```

Comment:
Can you move the logic to `setDefaultDisabledElements`? It introduces indirection but the reason why we did that is sometimes we need to conditionally disable an element based on the constructor options.

Reply:
I moved the logic to `setDefaultDisabledElements`